.eslintrc.json

```diff
@@ -0,0 +1,41 @@
+{
+  "env": {
+    "browser": true,
+    "node": true,
+    "es6": true,
+    "jest": true
```

Comment:
This testing tool is not required for our startup empty project, so remove it.

Reply:
Thanks @Emmanuel-Dominic for your feedback. However, this is not a testing tool. I was specifying environments. If you want to add testing tools you declare them in package.json.
Use this resource on how to configure eslint to gain a more understanding.
https://eslint.org/docs/user-guide/configuring